examples/StreetlightsAPI/StreetlightsAPI.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Can we keep the example version on 3.1 - I want to keep the examples using the latest stable version as this will be the default for new projects.